cycle_for/task_04.py

```diff
@@ -0,0 +1,7 @@
+# Дано 10 целых чисел. Вычислите их сумму. Напишите программу, использующую наименьшее число переменных.
```

Comment:
As the task says - you should use the minimal number of variables. You can still reduce it further like this:
```
sum = 0
for i in range(10):
    sum += int(input())
print(sum)
```